app/Http/Controllers/Api/ReviewsController.php

```diff
@@ -43,6 +43,8 @@ class ReviewsController extends ApiController
      */
     public function __construct(PostService $posts, ReportbackService $reportbacks)
     {
+        $this->middleware('api');
```

Comment:
@DFurnes i thought it was weird that we had to leave routes out of the `v2` group because we didn't want the api middleware to apply.